### PR TITLE
Sorting: add outlier detection as a sorting option

### PR DESCRIPTION
**Pull request by matyax**
This PR adds outlier detection to sort value breakdowns. It works by using [DBSCAN](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/forecasts/outlier-detection/#select-the-detection-algorithm-and-sensitivity) to detect outlying intervals, and sorting by the detected quantity.

Addtionally, we're wrapping all the WASM invocations, logging the errors and falling back to StdDev when it happens.

### CODE REVIEW

File: src/Components/ServiceScene/Breakdowns/SortByScene.tsx [hunk 1]
```diff
@@ -28,6 +28,11 @@ export class SortByScene extends SceneObjectBase<SortBySceneState> {
           label: 'Most relevant',
           description: 'Smart ordering of graphs based on the most significant spikes in the data',
         },
+        {
+          value: 'outliers',
+          label: 'Detected outliers',
+          description: 'Order by detected outliers in the data',
```
Comment (svennergr) [inline src/Components/ServiceScene/Breakdowns/SortByScene.tsx:34]: What does that mean, btw? Is it more "Order by the count/amount of outliers"?

**Comment by svennergr** [comment]
I think we need to also gate https://github.com/grafana/explore-logs/blob/85085a9643def5e8ecb103aa43faa2ea8abcd971/src/module.ts#L8 to only init when wasm is supported.

File: src/Components/ServiceScene/Breakdowns/SortByScene.tsx (hunk 1 above)
Comment (matyax) [reply src/Components/ServiceScene/Breakdowns/SortByScene.tsx:34]: It's a good question. Let's add more details to the description.

**Comment by matyax** [comment]
Right! Great idea, done.

**Review by gtk-grafana — APPROVED** [review]
LGTM and working well locally

**Pull request merged**